openstudiocore/src/model/ThermalZone.cpp

```diff
@@ -541,12 +541,28 @@ namespace detail {
   }
 
   bool ThermalZone_Impl::setFractionofZoneControlledbyPrimaryDaylightingControl(double fractionofZoneControlledbyPrimaryDaylightingControl) {
+
+    double secondaryFrac = fractionofZoneControlledbySecondaryDaylightingControl();
```

Comment:
@jmarrec these fraction controlled fields are currently set in the inspector which bypasses this logic, so i think this check needs to be implemented in the forward translator as well.  My approach would be if the sum is > 1, set the secondary fraction to 1-primary fraction.  You could do that in the translator as well as in these set fraction controlled methods.  What do you think?  It is a little goofy having one setter change another field, but it also seems like you would want the logic to be consistent between the API and translator